src/data/player.py

```diff
@@ -230,13 +231,13 @@ def league_tuple(self) -> LeagueTuple:
     def federation_tuple(self) -> FederationTuple:
         return FederationTuple(self.federation)
 
-    def compute_points(self, max_round: int):
-        """Computes and stores the points of the player,
-        from round 1 to round `max_round` (returns None)"""
+    def compute_points(self, max_round: int) -> float:
```

Comment:
Not sure it's correct to change the API like this.
I think you should add a helper function (named something like `points_before`) and use it in `compute_points`